doc/developers/kv/kv_serialisation.rst

```diff
@@ -3,12 +3,12 @@ KV Serialisation
 
 Every transaction executed by the primary on its key-value store is serialised before being replicated to all backups of the CCF network and written to the ledger.
 
-Transactions on private :cpp:class:`kv::Map` are encrypted before being serialised while transactions on public :cpp:class:`kv::Map` are only integrity-protected and readable by anyone with access to the ledger.
+Writes to private :cpp:class:`kv::Map`\s are encrypted before being serialised, and can only be decrypted by other nodes within the service. Writes to public :cpp:class:`kv::Map`\s are only integrity-protected; they are readable by anyone with access to the ledger.
```

Comment:
I suggest replacing "serialised" with "written to the ledger" here to avoid confusion.